server/shared/types/routes/hicdata.ts

```diff
@@ -1,4 +1,7 @@
 export type HicdataRequest = {
+	/** specify whether observed, expected, or observed/expected(o/e) values.  */
```

Comment:
use this new description:
Value is the 1st parameter of straw tool, and can only use these specific strings but not anything else